tests/bud.bats

```diff
@@ -366,6 +366,26 @@ _EOF
   expect_output --substring "invalid response status"
 }
 
+@test "build test has gid in supplemental groups" {
+  _prefetch alpine
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
+  run_buildah build $WITH_POLICY_JSON -t source -f $BUDFILES/supplemental-groups/Dockerfile
+  # gid 1000 must be in supplemental groups
+  expect_output --substring "Groups:	1000"
+}
+
+@test "build test has gid in supplemental groups and --isolation chroot" {
+  skip_if_rootless_environment
+  skip_if_no_runtime
+  test -z "${BUILDAH_ISOLATION}" || skip "BUILDAH_ISOLATION=${BUILDAH_ISOLATION} overrides --isolation"
+
+  _prefetch alpine
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
```

Comment:
What's this directory used for?

Reply:
Nothing, I had a inline file in first draft but later removed it so this is not needed.